product_sequence_categ/__openerp__.py

```diff
@@ -0,0 +1,38 @@
+# -*- encoding: utf-8 -*-
+###############################################################################
+# #
+# Copyright (C) 2015  KMEE  - www.kmee.com.br - Rodolfo Bertozo               #
+#                                                                             #
+# This program is free software: you can redistribute it and/or modify        #
+# it under the terms of the GNU Affero General Public License as published by #
+# the Free Software Foundation, either version 3 of the License, or           #
+# (at your option) any later version.                                         #
+#                                                                             #
+# This program is distributed in the hope that it will be useful,             #
+# but WITHOUT ANY WARRANTY; without even the implied warranty of              #
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the               #
+# GNU General Public License for more details.                                #
+#                                                                             #
+# You should have received a copy of the GNU General Public License           #
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.       #
+###############################################################################
+
+{
+    'name': 'Product Sequence Category',
+    'version': '0.1',
+    'category': 'Generic Modules',
+    'description': """Generate product sequence reference based on sequence from category""",
```

Comment:
It's a description like this? 
https://github.com/OCA/product-attribute/blob/8.0/product_brand/README.rst

Is there any guide about how to contribute? Is this http://odoo-community.org/page/how-to?